### ISSUE TITLE
Error 'The method or operation is not implemented' when trying to get the supported languages for OCR in Android

### ISSUE THREAD

**Issue by geertgeerits** [kfrancis/ocr#7]
Hi, thank you for this plugin.

Getting the supported languages works in windows and iOs but not in Android.
Maybe I'm doing something wrong.
This is my code:

```
private List<string>? supportedLanguages;

        protected async override void OnAppearing()
        {
            base.OnAppearing();
            await OcrPlugin.Default.InitAsync();
            InitializeSupportedLanguages();
        }

        private async void InitializeSupportedLanguages()
        {
            try
            {
                supportedLanguages = OcrPlugin.Default.SupportedLanguages.ToList();

                if (supportedLanguages.Count > 0)
                {
                    foreach (var language in supportedLanguages)
                    {
                        Debug.WriteLine(language);
                    }
                }
                else
                {
#if DEBUG
                    await DisplayAlert("Error", "No supported languages found", "OK");
#endif
                }
            }
            catch (Exception ex)
            {
#if DEBUG
                await DisplayAlert("Error", ex.Message, "OK");
#endif
            }          
        }

```

**Comment by kfrancis** [comment]
No, that's correct. I don't know of a way to get supported languages like it's made available in ios.

**Comment by kfrancis** [comment]
I've released v1.0.12 with that update. I'm not entirely sure about the language list, I think it's right based on the documentation but if you do see something - feel free to put in a pull request :)
